### PR TITLE
user/papers: new package

### PR DESCRIPTION
**Pull request by tulilirockz**
GNOME papers and its deps!

### CODE REVIEW

File: user/papers/template.py [hunk 1]
```diff
@@ -0,0 +1,63 @@
+pkgname = "papers"
+pkgver = "46.2"
+pkgrel = 0
+build_style = "meson"
+configure_args = [
+    "-Dcomics=enabled",
```
Comment (nekopsykose) [inline user/papers/template.py:6]: you can documentation=false and then don't need gi-docgen at all

Comment (tulilirockz) [reply user/papers/template.py:6]: LOL no way

File: user/papers/patches/tests.patch [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+   install_dir: join_paths(pps_datadir, 'metainfo'),
+ )
+-
+-# Validate Appdata
+-if get_option('tests')
+-  appstreamcli = find_program('appstreamcli', required: true)
+-  if (appstreamcli.found())
+-    test('validate-appdata',
+-         appstreamcli,
+-         args: ['validate', '--strict', '--pedantic', '--explain', metainfo_file],
```
Comment (nekopsykose) [inline user/papers/patches/tests.patch:14]: you can probably add only `--nonet` to these args and it will work :)

File: user/papers/template.py [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+    cargo.Cargo(self, wrksrc="shell-rs").vendor()
+
+
+def init_build(self):
+    from cbuild.util import cargo
+
+    renv = cargo.get_environment(self)
+    self.make_env.update(renv)
+
+
```
Comment (nekopsykose) [inline user/papers/template.py:55]: i think you also need a -devel package given it has libraries and a usr/include

also maybe a -nautilus (see font-manager subpackage and copy that)

File: user/papers/patches/tests.patch (hunk 2 above)
Comment (tulilirockz) [reply user/papers/patches/tests.patch:14]: omg true

**Pull request merged**